README.md

```diff
@@ -101,74 +31,98 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+provider "ibm" {
+  ibmcloud_api_key = ""
```

Comment:
There was no need to remove what you had here. You can just re-audit the baseline and mark it as not a secret:

`detect-secrets scan --update .secrets.baseline`
`detect-secrets audit .secrets.baseline`